script/travis/push_doc.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+#
+# Copyright 2020 Benjamin Worpitz
+#
+# This file is part of Alpaka.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+
+source ./script/travis/travis_retry.sh
+
+source ./script/travis/set.sh
+
+cd doc/doxygen/html
+
+git config --global user.email "action@github.com"
+git config --global user.name "GitHub Action"
+
+git add .
+
+git log -n 3
+
+git diff --quiet && git diff --staged --quiet || (git commit -m "Update documentation skip-checks: true"; git push origin gh-pages)
+
+cd ../../../
```

Comment:
Github shows no empty line at the end. Same for another `.sh` file.

Reply:
fixed